### PR TITLE
feat(rpc): Added bedrock

### PR DESCRIPTION
**Pull request by qbzzt**
Closing: DOCS-588

Asked for engineering review because I'm not sure that it's non-fiction.

### CODE REVIEW

File: src/docs/developers/build/json-rpc.md [hunk 1]
```diff
@@ -179,3 +182,137 @@ Optimism nodes also block the `eth_sendTransaction` method for the same reasons
...
+The execution engine also has a secondary RPC interface, on another port, implementing the [engine API](https://github.com/ethereum/execution-apis/blob/main/src/engine/specification.md) for use by the rollup node. 
+This API has [a few changes](https://github.com/ethereum-optimism/optimism/blob/develop/specs/exec-engine.md#engine-api) for the Optimism rollup environment. 
+
+Optionally, the execution engine can provide [snap sync](https://github.com/ethereum/devp2p/blob/master/caps/snap.md) functionality to other execution engines.
+
+### Custom methods
+
+These custom methods provide rollup specific information.
+
+#### `rollup_getInfo`
```
Comment (mslipper) [inline src/docs/developers/build/json-rpc.md:226]: These methods no longer exist on `op-geth`.

File: src/docs/developers/build/json-rpc.md [hunk 2]
```diff
@@ -179,3 +182,137 @@ Optimism nodes also block the `eth_sendTransaction` method for the same reasons
...
+      "queueIndex":12481,
+      "verifiedIndex":0
+    }
+  }
+}
+```
+
+---
+
+#### `rollup_gasPrices`
```
Comment (mslipper) [inline src/docs/developers/build/json-rpc.md:275]: Method no longer exists.

Comment (qbzzt) [reply src/docs/developers/build/json-rpc.md]: Oops, I meant to remove them from there, but didn't.  Fixed now.

**Comment by qbzzt** [comment]
My bad, I resolved them in a PR 5 days ago, but didn't mark that.

**Review by Nicca42 — APPROVED** [review]

**Pull request merged**